packages/markdown-it-container/package.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Copied from fork, small changes such as package name and using `pnpm` instead of `npm`